src/main/java/com/hazelcast/kubernetes/RetryUtils.java

```diff
@@ -65,6 +64,13 @@ private RetryUtils() {
         }
     }
 
+    private static RuntimeException rethrowUnchecked(Exception e) {
```

Comment:
Well, it doesn't really _rethrow_ anything, so it might be worth adjusting the name :)

Reply:
👍 

Updated. I also updated the same for hazelcast-aws.